features/step_definitions/auction_steps.rb

```diff
@@ -35,13 +61,17 @@
 end
 
 When(/^I visit the auction page$/) do
-  visit "/auctions/#{@auction.id}"
+  visit(auction_path(@auction.id))
 end
 
 When(/^I visit the unpublished auction$/) do
   visit "/auctions/#{@unpublished_auction.id}"
 end
 
+When(/^I visit the auction bids page$/) do
+  visit(auction_bids_path(@auction.id))
```

Comment:
protip: you can pass `@auction` rather than `@auction.id` when using named routes like this. Rails automatically calls `to_param` on any object passed into a named routed, and the default `to_param` def returns `id`